src/stdlib/map.md

```diff
@@ -145,3 +145,19 @@ fromArray : Array<(k, v)> -> Map<k, v>
 ```
 
 Creates a map from an array of key-value pairs.
+
+### Map.**filter**
+
+```grain
+filter : ((k, v) -> Bool, Map<k, v>) -> Void
+```
+
+Keeps all key-value pairs that the predicate returned `true` for.
```

Comment:
```suggestion
Removes all key-value pairs that the predicate returns `false` for from the map.
```

Reply:
My reasoning for the wording was that `filter` is considered a "positive" action, if that makes sense.
Another example is ReasonML's API - they use the term `keep` instead of `filter`.

I just wanted to explain myself and bottom line this is your call, I don't really have strong feelings about it :)

Reply:
That's reasonable! Let's just add `from the map` to the end 🙂